functions/check-started/index.js

```diff
@@ -91,3 +97,26 @@ async function updateAdminDatabaseForCheckStarted (checkCode, startedAt, logger)
     throw error
   }
 }
+
+/**
+ * Fetch check record based on check code
+ * @param {String} checkCode - the unique GUID that identifies the check in the admin DB
+ * @param {Function} logger
+ * @return {Object} check record
+ */
+async function sqlFindCheckByCheckCode (checkCode, logger) {

```

Comment:
You don't need to write this function again - just import '../lib/sql-helper' into the function.  There is an example usage like this in completed-check.

Reply:
thanks @jon-shipley, didn't spot that helper. I have updated based on that recommendation.